### PR TITLE
Update pegasus_de.ts

### PR DESCRIPTION
**Pull request by TigraTT-Driver**
fixed some typos and translations. Also updated to the newest en template

### CODE REVIEW

File: pegasus_de.ts [hunk 1]
```diff
@@ -34,114 +34,159 @@ in den folgenden Verzeichnissen:</translation>
...
+(timeout in %1 seconds)</source>
+        <translation>Zum Festlegen eine beliebige Achse neigen
+        (Zeitüberschreitung in %1 Sekunden)</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadRecorder.qml" line="108"/>
+        <source>Press any button now to record
+(timeout in %1 seconds)</source>
+        <translation>Zum Festlegen eine beliebige Taste drücken
+        (Zeitüberschreitung in %1 Sekunden)</translation>
```
Comment (mmatyas) [inline pegasus_de.ts]: Make sure there's no leading space in the `(Zeitüberschreitung in %1 Sekunden)` lines.

**Review by mmatyas — COMMENTED** [review]
Thanks! Just one little thing I've noticed:

File: pegasus_de.ts (hunk 1 above)
Comment (TigraTT-Driver) [reply pegasus_de.ts]: sry for the inconvenience. I've updated it and checked if there are other wrong leading spaces

**Comment by mmatyas** [comment]
Thanks!

**Pull request merged**